src/gtk/exo/exo-icon-view.c

```diff
@@ -1878,6 +1878,16 @@ exo_icon_view_expose_event (GtkWidget      *widget,
 
   if (!gtk_cairo_should_draw_window (cr, priv->bin_window))
     return FALSE;
+
+  /* draw a background according to the css theme */
+  style = gtk_widget_get_style_context (widget);
```

Comment:
I suppose it should be not retrieved again at line 1993 with this addition, right?

Reply:
Correct, I just tried and it works the same. PR updated